presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -310,6 +310,13 @@ Property Name                                      Description
                                                    can be used to use a different location for each user.
 
 ``hive.translate-hive-views``                      Enable translation for Hive views. (experimental)            ``false``
+
+``hive.legacy-hive-view-translation``              Use the legacy algorithm to translate Hive views.            ``false``
+                                                   Set this property to ``true`` if you encounter unhandled 
+                                                   AST token exceptions with the new experimental algorithm. 
```

Comment:
This is not limited to AST token expcetions (eg https://github.com/prestosql/presto/issues/6234), so
it should be at least `e.g.`, but i think we can remove the "troubleshooting" advice from here
cc @losipiuk

Reply:
i see know Łukasz suggested the same in https://github.com/prestosql/presto/pull/6261#discussion_r539202967